src/service/utils/viewConnection.js

```diff
@@ -188,7 +188,7 @@ const setCanAutoPlayStream = async () => {
       muted: state.Params.viewer.muted,
     })
     const muted = !state.Params.viewer.muted
-      ? !canAutoPlayVideo.result
+      ? canAutoPlayVideo.result
```

Comment:
This previously would always return true because the query param and canAutoPlayVideo response cancelled each other out. 

![checkValues1](https://github.com/user-attachments/assets/72044b6f-1dfa-46f1-9de9-1aa7a45ee575)
![CheckValues](https://github.com/user-attachments/assets/7b63c253-f959-4d79-bf01-7a2674178950)